src/main/resources/db/migration/V12__new_table_failed_groups.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE failed_groups (
```

Comment:
Do you think that `failed_device_groups` would be a more precise name? Applies to class/method names below as well.

Reply:
On the one hand yes, I think it would be more precise. On the other hand, all over dev-reg and campaigner there is just the "group" concept to represent "device group". So it would kinda break the convention I guess. But I have no strong opinion about this, I can rename it.

Reply:
Ah, true, it is only `group` indeed... Yeah, I guess in this case it's better to leave it as it is now.